docs/src/main/paradox/slick.md

```diff
@@ -158,6 +158,8 @@ Scala
 Java
 : @@snip [snip](/slick/src/test/java/docs/javadsl/DocSnippetFlow.java) { #flow-example }
 
+Java (JDBC)
```

Comment:
Please remove the old Java example. It is very bad to show this dangerous way of construction SQL statements.

Reply:
If we want not to show this approach in docs, maybe needly also to mark these API as deprecated? WDYT?

Reply:
It is still fine if the user knows what they are doing. Eg. when the data is properly typed or validated.
We can leave it as is, but not recommend it.

Reply:
✅ Done